frontend/src/components/Login/LoginSidePage.css

```diff
@@ -0,0 +1,10 @@
+.gradient-custom-side {
```

Comment:
Are LoginPage and LoginSidePage different designs to be suggested to client?

Reply:
Those 2 components are each 1/2 of the whole login screen, <LoginPage/> is actually a component with inputs, that's on the left side on the screen, and <LoginSidePage /> is just a component on the right side with centered text that goes away when the screen size is smaller.

We will make a few more designs :)